tests/gradients/test_parameter_shift.py

```diff
@@ -359,11 +399,6 @@ def test_grad_recipe_parameter_dependent(self, monkeypatch):
         shift rule. Also tests that grad_recipes supersedes paramter_frequencies.
         """
 
-        def fail(*args, **kwargs):
-            raise qml.operation.ParameterFrequenciesUndefinedError
-
-        monkeypatch.setattr(qml.RX, "parameter_frequencies", fail)
-

```

Comment:
`parameter_frequencies` do not take precedence over `grad_recipe`, which is modded here. Therefore we do not need to deactivate `parameter_frequencies` to activate the `grad_recipe` mod. 
In particular the sentence "Also tests that grad_recipes supersedes parameter_frequencies" in the docstring seems to be incorrect as far as I can tell (Superseding would be checked if `parameter_frequencies` existed but were not used).

Reply:
Should the docstring be updated then?

Reply:
I guess now it is actually correct :D Because we do not deactivate parameter_frequencies and check whether `grad_recipe` is used with priority over it.